src/managed/Microsoft.Extensions.DependencyModel/ApplicationEnvironment.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
License doesn't follow the new header (I realize this is a copy of the old code)... Applies to other files as well.

Reply:
This is tracked by https://github.com/dotnet/core-setup/issues/5224. I see there are 100+ files in this repo that need the update.